experimental/README.md

```diff
@@ -45,29 +45,26 @@ refined or entirely removed.
 
     This command downloads a test image and runs it in a container.
 
-### Get the Linux binary
-To download the latest experimental `docker` binary for Linux,
-use the following URLs:
+4. And verify the experimental flag is enabled:
 
-    https://experimental.docker.com/builds/Linux/i386/docker-latest.tgz
+        $ docker version -f '{{.Server.Experimental}}'
+        true
 
-    https://experimental.docker.com/builds/Linux/x86_64/docker-latest.tgz
+## Use Docker experimental with the regular package
 
-After downloading the appropriate binary, you can follow the instructions
-[here](https://docs.docker.com/engine/installation/binaries/#/get-the-docker-engine-binaries) to run the `docker` daemon.
+Starting with Docker 1.13, you can use the experimental features even with the regular package.
+For enabling experimental features, you need to start the Docker daemon with
+`--experimental` flag.
+You can also enable the daemon flag via `/etc/docker/daemon.json`. e.g.
 
-> **Note**
->
-> 1) You can get the MD5 and SHA256 hashes by appending .md5 and .sha256 to the URLs respectively
->
-> 2) You can get the compressed binaries by appending .tgz to the URLs
+        {
+            "experimental": true
+        }
 
-### Build an experimental binary
-You can also build the experimental binary from the standard development environment by adding
-`DOCKER_EXPERIMENTAL=1` to the environment where you run `make` to build Docker binaries. For example,
-to build a Docker binary with the experimental features enabled:
+Then make sure the experimental flag is enabled:
 
-        $ DOCKER_EXPERIMENTAL=1 make binary
```

Comment:
I removed this line because `make binary` doesn't produce `daemon.json` with the `experimental` flag.